### PR TITLE
Refactor build.sbt to use inline dependencies 

### PR DESCRIPTION
**Pull request by Reettaphant**
Refactor build.sbt to use inline dependencies so that snyk can understand them and identify vulnerabilities correctly.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -24,15 +70,81 @@ lazy val common = (project in file("common"))
...
   )
 
 lazy val app = (project in file("."))
   .dependsOn(common)
   .enablePlugins(PlayScala, SbtWeb, BuildInfoPlugin, JDebPackaging)
   .settings(commonSettings,
     name := "media-atom-maker",
-    libraryDependencies ++= Dependencies.appDependencies,
+    libraryDependencies ++= Seq(
+      "com.gu" %% "pan-domain-auth-play_2-5" % pandaVersion,
```
Comment (mbarton) [inline build.sbt]: Where we already have the dependency in `common` we don't need to re-state it here

**Comment by franziskas** [comment]
@Reettaphant @mbarton is it worth revisiting this PR?

**Comment by Reettaphant** [comment]
@franziskas @mbarton Yes, worth revisiting, I got the tests passing yesterday and this is now ready to go.

**Comment by akash1810** [comment]
This seems to have broken the integration tests.

**Pull request merged**